### ISSUE TITLE
Awesome library! Any way to change the color of the WHOLE bar between a specific interval?

### ISSUE THREAD

**Issue by sjl2024** [kottenator/jquery-circle-progress#65]
E.g. circle is RED when between 0-.33, yellow between .33-66, so on, so forth.

Thanks again!

**Comment by kottenator** [comment]
Please, take a look at [this ticket](http://jsbin.com/guteko/4).

If you want to have exact 3 colors without smooth color transition, you can easily modify the event listener to check if the value is inside one of three intervals and set the correspondent color.

**Comment by sjl2024** [comment]
amazing. thanks so much for that.

sorry if this has already been asked - how do I 'append' progress to the circle incrementally? Say I want a smooth transition from 0 to 1 over 10 seconds, or something along those lines.

**Comment by kottenator** [comment]
[Here is a snippet for you](http://jsbin.com/pubuya/3/edit?js,output). Does it answer all your questions? :)

**Comment by sjl2024** [comment]
I think so! Keeping open just in-case! Thank you SO MUCH!

**Comment by sjl2024** [comment]
It seems like your strategy for continuous movement (interval) doesn't play friendly with setting the arcFill between certain values.

```
      c4.circleProgress({
          thickness: 12,
          startAngle: -Math.PI / 2
      }).on('circle-animation-progress', function(e, p, v) {
          var $this = $(this),
            instance = $this.data('circle-progress');
          
          //instance.arcFill = 'hsl(' + 110 * v + ', 50%, 75%)';
          
          if (p > 0 && p < .33) {
            instance.arcFill = curArcColor = '#333';
          }
          if (p > .33 && p < .66) {
            instance.arcFill = curArcColor = '#666';
          }
          else {
            instance.arcFill = curArcColor = '#999';
          }
      });
```

Also, what is the difference between:

```
c4.circleProgress('value', i / 10);
```

and 

```
      c4.circleProgress({
          value: i/10,
      });
```

One (the first) appends progress and the second forces the animation to start over (even though both use i/10 as the value).

**Comment by kottenator** [comment]
Yes, you're right, first updates the value, second resets the value and restarts the animation.

The idea is that when you do `el.circleProgress({ smth: other })`, you re-use previously created circle, change _`smth` property_ (other properties will be preserved) and restart the animation.

**Comment by hioctane321** [comment]
> Please, take a look at [this ticket](http://jsbin.com/guteko/4).
> 
> If you want to have exact 3 colors without smooth color transition, you can easily modify the event listener to check if the value is inside one of three intervals and set the correspondent color.

Link broken?
